core/src/main/java/org/web3j/protocol/rx/JsonRpc2_0Rx.java

```diff
@@ -81,8 +81,8 @@ public JsonRpc2_0Rx(Web3j web3j, ScheduledExecutorService scheduledExecutorServi
 
     public Observable<Transaction> pendingTransactionObservable(long pollingInterval) {
         return ethPendingTransactionHashObservable(pollingInterval)
-                .flatMap(transactionHash ->
-                        web3j.ethGetTransactionByHash(transactionHash).observable())
+                .flatMap(transactionHash -> web3j.ethGetTransactionByHash(transactionHash).observable())
```

Comment:
Build fails as the line is too long. Please revert the change you made here.

Reply:
Better with https://github.com/web3j/web3j/pull/430/commits/fcb7ef8a64fe2a5d0c38de9498b09106ce884a47 ?